src/UnityExtension/Assets/Editor/GitHub.Unity/UI/AuthenticationView.cs

```diff
@@ -134,35 +134,42 @@ private void OnGUILogin()
             GUILayout.Space(3);
             GUILayout.BeginHorizontal();
             {
-                if (busy) GUI.enabled = false;
-                username = EditorGUILayout.TextField(usernameLabel ,username, Styles.TextFieldStyle);
-                GUI.enabled = true;
+                EditorGUI.BeginDisabledGroup(isBusy);
```

Comment:
I will look at this as well, didn't want to change scope before, as I didn't know what you guys wanted.